static/bidder-info/visiblemeasures.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://us-e.visiblemeasures.com/pserver"
+maintainer:
+  email: "Support@visiblemeasures.com"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+    
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://cs.visiblemeasures.com/pserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
redirect url returns 404 status response

url: https://cs.visiblemeasures.com/pserver?gdpr=&gdpr_consent=&us_privacy=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dvisiblemeasures%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D
https://prebid.adnxs.com/pbs/v1/setuid?bidder=visiblemeasures&gdpr=&gdpr_consent=&f=i&uid=[UID]

<img width="1792" alt="Screenshot 2023-02-09 at 2 08 25 PM" src="https://user-images.githubusercontent.com/24757781/217760178-2a907cc3-7d9a-4786-82a6-26201baf29da.png">

Reply:
It fixed, should works now